### ISSUE TITLE
Un-zip files insdie owncloud

### ISSUE THREAD

**Issue by dtb1987** [owncloud/apps#2140]
I would like to be able to unzip files in owncloud.

A lot of the time i zip files to upload them but later i might only want to download one specific file from that zipped folder, in this case it would be helpful to be able to unzip it within owncloud

**Comment by e-alfred** [comment]
This enhancement should be extended to rar, 7zip, tar and other archive types as well.

**Comment by PatelParas** [comment]
This one was discussed previously on following thread : https://github.com/owncloud/core/issues/8378

**Comment by dtb1987** [comment]
Not really, they wanted an app to do it, i would like to see this built in as a feature
